python/lsst/ts/xml/data/sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -215,4 +215,16 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTDome</Subsystem>
+    <EFDB_Topic>MTDome_command_setPowerManagementMode</EFDB_Topic>
+    <Description>Set the power management mode.</Description>
+    <item>
+      <EFDB_Name>powerManagementMode</EFDB_Name>
```

Comment:
consider renaming this field to `mode` only. The command is already "setPowerManagmentMode" there is no need to repeat "powerManagement" here.